SPECS-EXTENDED/rust-coreutils/rust-coreutils.spec

```
@@ -0,0 +1,150 @@
+Summary:        Basic system utilities; reimplemented in Rust
+Name:           rust-coreutils
+Version:        0.0.26
+Release:        2%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/uutils/coreutils
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Below is a automatically created tarball with no download link.
+Source1:        rust-coreutils-0.0.26-vendored.tar.gz
+Source2:        cargo_config.toml
+
+
+BuildRequires:  cargo
+BuildRequires:  gcc
+BuildRequires:  glibc
+Conflicts:      coreutils
+
+%description
+This package provides the reimplementation of the GNU core utilities in Rust.
+
+%prep
+%setup -q -n coreutils-%{version}
+tar --strip-components=1 -xzf %{SOURCE1}
```

Suggestion: can we build the vendored tarball, so that we don't need `--strip-components=1`? We can also switch to `%autosetup` and end up with this:
```suggestion
%autosetup -q -a 1 -n coreutils-%{version}
```